README.md

```diff
@@ -1,19 +1,197 @@
-# logging
+# Fluent Bit output plugin
 
-[![Go Report Card](https://goreportcard.com/badge/github.com/gardener/logging)](https://goreportcard.com/report/github.com/gardener/logging)
+This plugin extends [Grafana,s Fluent Bit output plugin](https://github.com/grafana/loki/tree/master/cmd/fluent-bit) which aims to forward log messages from fluent-bit to Loki.
```

Comment:
Add link here

Reply:
the link must contain specific version not the master branch

Reply:
It is still pointing to the master branch